torchtune/models/clip/_model_builders.py

```diff
@@ -1,4 +1,63 @@
-from torchtune.models.clip._transforms import CLIPImageTransform
```

Comment:
What belongs in the `_model_builders` and what belongs in the `_component_builders`? It seems like we generally put model and tokenizer builders in `_model_builders` for most models like qwen, etc. But for CLIP vision, it's flipped (model builders are in `_component_builders` and the transform is in `_model_builders`). In fact, this file was broken because this import was misnamed, so I assume this file wasn't even really being used.